ReleaseHistory.md

```diff
@@ -1,10 +1,11 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
-## UNRELEASED
+## 4.6.0 UNRELEASED
 * BRK: Remove defunct and unsupported `kusto` command in `Sarif.Multitool`.
 * DEP: Remove dependency on `Microsoft.Azure.Kusto.Data`.
-* DEP: Update `Azure.Identity` reference from 1.10.2 to 1.12.1 in `WorkItems` and `Sarif.Multitool.Library` to resolve [CVE-2024-29992](https://github.com/advisories/GHSA-wvxc-855f-jvrv) and other CVEs.
-* DEP: Update `Azure.Core` from 1.35.0 to 1.41.0 to satisfy minimum requirement of `Azure.Identity` 1.12.1 that also has no vulnerabilities.
+* DEP: Update `Azure.Identity` reference from 1.10.2 to 1.13.1 in `WorkItems` and `Sarif.Multitool.Library` to resolve [CVE-2024-29992](https://github.com/advisories/GHSA-wvxc-855f-jvrv) and other CVEs.
```

Comment:
>DEP [](http://example.com/codeflow?start=2&length=3)

I also see a DEP upgrade, do we want to add in release note

`<PackageVersion Include="System.Text.Encodings.Web" Version="5.0.1" />`
==>
    `<PackageVersion Include="System.Text.Encodings.Web" Version="6.0.0" />`

Reply:
Yes, good catch, thanks! Btw, Shaopeng, do you understand why our ubunto test automation is failing?